packages/create/create.js

```diff
@@ -85,6 +85,12 @@ class LineSplitter extends stream.Transform {
 class InstallError extends Error { }
 
 
+async function safeOutputFile(file, data, options={}) {
```

Comment:
Why is it defined here, can't it be just imported from the file_ops like it is for the rest of the files?

Reply:
create runs by npm downloading all of its dependencies, unpacking them into a temporary directory and then invoking it from there. It doesn't have `@clusterio/lib` as a dependency because that's way too big to recursively download all dependencies of and run on the fly.